src/integrationtests/java/util/IntegrationTestBase.java

```diff
@@ -153,13 +154,19 @@ protected void sendEventsAndConfirm() throws Exception {
                     TimeUnit.MILLISECONDS.sleep(firstEventWaitTime);
                     alreadyExecuted = true;
                 }
+                /**
+                 * Without a small delay between the sending of 2 events, one may risk to be lost in
+                 * an empty void of not received by EI
+                 */
+                TimeUnit.MILLISECONDS.sleep(DEFAULT_DELAY_BETWEEN_SENDING_EVENTS);
             }
 
-            // wait for all events to be processed
             waitForEventsToBeProcessed(eventsCount);
             checkResult(getCheckData());
         } catch (IOException e) {
-            LOGGER.error(e.getMessage(), e);
+            String message = String.format("Failed to send Eiffel messages. Reason: %s", e.getMessage());
```

Comment:
I think this is a bit redundant and it was previously as well. When the logger prints e, it will print the message and the stack trace.

Reply:
I reuse message that is used to inform the tester what failed.